tests/manage/pv_services/test_overprovision_level_policy_control.py

```diff
@@ -33,8 +38,9 @@ def setup_sc(storageclass_factory_class):
 
 @tier1
 @bugzilla("2024545")
-@pytest.mark.polarion_id("OCS-4472")
+@skipif_external_mode
```

Comment:
@OdedViner I see this skip marker added here.
Are these tests not applicable for external mode?

Reply:
I have some issues with parameters on External cluster

Reply:
If the tests are not applicable for external mode, then it make sense to add this marker.
Otherwise, I think @red-hat-storage/ocs-qe-ecosystem can help with the external mode cluster.